best-practices/native/react-native/README.md

```diff
@@ -13,7 +13,8 @@ There are typically 2 options for React Native development:
     * You should read the [caveats](https://facebook.github.io/react-native/docs/getting-started#caveats) on using Expo, but for Shift3, the main drawbacks are:
         1. If your project will need background support (the app should keep running when not in focus) DO NOT use Expo 
 2. [Native Code](startup/native-code.md)
-    * This is a more difficult project to start up, but you have more flexibility once it's started that with Expo. 
+    * This is a more difficult project to start up, but you have more flexibility once it's started that with Expo.
```

Comment:
"than with Expo."

Reply:
Grammar will be the death of me. Thanks, ill make the changes.

Reply:
fixed via: 6332389b5f20bb6165be0110ea918a57634cc5ce